scheduler/scripts/run_main.py

```diff
@@ -31,8 +31,8 @@
 
 def main(*,
          verbose: bool = False,
-         start: Optional[Time] = Time("2018-10-01 08:00:00", format='iso', scale='utc'),
-         end: Optional[Time] = Time("2018-10-03 08:00:00", format='iso', scale='utc'),
+         start: Optional[Time] = Time("2018-08-01 08:00:00", format='iso', scale='utc'),
+         end: Optional[Time] = Time("2018-08-03 08:00:00", format='iso', scale='utc'),
```

Comment:
Just curious why the change in dates here.

Also, the dates are kind of screwy: the original `2018-10-01` and `2018-10-03` actually schedules nights `2018-09-30` to `2018-10-02`, so if we do need the beginning of the semester, we should probably be doing `2018-08-02` rather than `2018-08-01` (which will probably calculate `2018-07-31`).